### PR TITLE
Added test for `ReactiveInputStream`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Test `ReactiveInputStream`

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@g4s8 please, could you take a look? Test with piped in/out streams is stack, I cannot figure out how to implement it properly.

File: src/test/java/org/cqfn/rio/stream/ReactiveInputStreamTest.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        InterruptedException {
+        try (
+            PipedOutputStream newout = new PipedOutputStream();
+            PipedInputStream newin = new PipedInputStream(newout)
+        ) {
+            final byte[] data = "xyz098".getBytes();
+            newout.write(data, 0, data.length);
+            MatcherAssert.assertThat(
+                this.single(
+                    new ReactiveInputStream(newin).read(Buffers.Standard.K1)
```
Comment (g4s8) [inline src/test/java/org/cqfn/rio/stream/ReactiveInputStreamTest.java]: @olenagerasimova try starting `ReactiveInputStream.read` before writing to pipe

Comment (olenagerasimova) [reply src/test/java/org/cqfn/rio/stream/ReactiveInputStreamTest.java]: Tried another variant, still stack

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Comment by olenagerasimova** [comment]
@g4s8 why have you merged it? The second it disabled, because it stacks

**Pull request merged**